zeppelin-zengine/pom.xml

```diff
@@ -257,6 +257,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.mongodb</groupId>
+      <artifactId>mongo-java-driver</artifactId>
+      <version>3.4.1</version>
```

Comment:
Thanks for the contribution.

While this PR introduces new dependencies that will be packaged and distributed in binary packages, 
license information of `mongo-java-driver` and all it's transitive dependencies need to be added to [zeppelin-distribution/src/bin_license/LICENSE](https://github.com/apache/zeppelin/blob/master/zeppelin-distribution/src/bin_license/LICENSE) file.

Following mvn command will print dependency tree with transitive dependencies

```
mvn [YOUR BUILD FLAGS] package dependency:tree
```

@tae-jun Could you take care of it? Let me know if you need any help!